src/Microsoft.Health.Fhir.Shared.Api/Features/Resources/Bundle/BundleHandler.cs

```diff
@@ -217,7 +229,17 @@ public async Task<BundleResponse> Handle(BundleRequest request, CancellationToke
                         Type = BundleType.TransactionResponse,
                     };
 
-                    return await ExecuteTransactionForAllRequests(responseBundle);
+                    var response = await ExecuteTransactionForAllRequests(responseBundle);
+
+                    int successfulRequestCount = 0;
+                    foreach (var entry in responseBundle.Entry)
+                    {
+                        successfulRequestCount += entry.Response.Status.StartsWith("2", StringComparison.InvariantCulture) ? 1 : 0;
+                    }
+
+                    await _mediator.Publish(new BundleMetricsNotification(successfulRequestCount, BundleType.Transaction), CancellationToken.None);
```

Comment:
Do/should we care about the request types within the bundle? i.e.
{ "read", 2 },
{ "update", 5 }

Reply:
I don't believe we will be billing differently for different types of calls. @CaitlinV39, is this correct?

Reply:
To correct my previous comment, we should group by response code:

{ "200", 5 },
{ "429", 1 },
{ "500", 1 }

A mapping for these codes is done in the health platform.